kubecolor.rb

```diff
@@ -0,0 +1,19 @@
+class Kubecolor < Formula
```

Comment:
This file feels out of place.

Accidentally committed? Or why is it here?

Reply:
removed.